src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -484,6 +484,8 @@ const generateTaskRole = (
                   'github.com/fuse': 1,
                   'nvidia.com/gpu':
                     config.taskRoles[taskRole].resourcePerInstance.gpu,
+                  'enflame.com/dtu':
```

Comment:
1. we don't have such card to test or use
2. this pr won't work and it will break all nodes

pls feel free to keep a fork to support your new card, and pls do test your code before submitting a pr